VerifyProfileEmail/handler.ts

```diff
@@ -102,8 +105,13 @@ export function VerifyProfileEmailHandler(
     );
 
     if (isLeft(errorOrMaybeVerificationTokenEntity)) {
+      const errorValue = errorOrMaybeVerificationTokenEntity.value;
+      const errorMessage = isArray(errorValue)
+        ? readableReport(errorValue)
+        : errorValue;
```

Comment:
this would be an `Error` object, shouldn't this be `errorValue.message`? btw see my other comment on functions-commons